packages/components/src/components/accordion/accordion.stories.ts

```diff
@@ -2,20 +2,27 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
-import cx from 'classix';
 
 const { argTypes, parameters } = storybookDefaults('sd-accordion');
 const { overrideArgs } = storybookHelpers('sd-accordion');
 const { generateTemplate } = storybookTemplate('sd-accordion');
 
+/**
+ *
+ * Accordions show a brief summary and expand to display additional content.
+ *
+ *  **Related templates**:
+ * - [Accordion Group](?path=/docs/templates-accordion-group--docs)
```

Comment:
* - [sd-accordion-group](?path=/docs/components-sd-accordion-group--docs)

Reply:
Not sure what you mean here. Can you please clarify?

Reply:
Clarified on teams call